### PR TITLE
Copy animation frame

### LINKED ISSUE #75: Add option to copy frames for Animation Drawables

### PR DESCRIPTION
**Pull request by tlein**
Resolves #75

### CODE REVIEW

File: src/duckling/controls/accordian.component.ts [hunk 1]
```diff
@@ -58,13 +66,31 @@ export class Accordian<T> {
...
+    @Output() elementCloned = new EventEmitter<T[]>();
 
     /**
      * Keeps track of what elements are currently opened
      */
     openedElements : {[key : string] : boolean} = {};
 
+    private indiceTogglesToProcess : {index : number, open : boolean}[] = [];
+
+    ngOnChanges(changes : {[key : string] : SimpleChange}) {
```
Comment (jeffswenson) [inline src/duckling/controls/accordian.component.ts]: You should remove the processIndexStateChanges logic.

File: src/duckling/controls/accordian-element.component.ts [hunk 2]
```diff
@@ -24,6 +24,13 @@ import {Icon} from './icon.component';
             <div class="right-buttons">
                 <button
                     md-button
+                    *ngIf="clone"
+                    class="display-on-hover"
```
Comment (jeffswenson) [inline src/duckling/controls/accordian-element.component.ts:28]: You should add a tooltip!

File: src/duckling/controls/accordian.component.ts (hunk 1 above)
Comment (tlein) [reply src/duckling/controls/accordian.component.ts]: fixed

File: src/duckling/controls/accordian-element.component.ts (hunk 2 above)
Comment (tlein) [reply src/duckling/controls/accordian-element.component.ts:28]: fixed

**Pull request merged**